### PR TITLE
[FIX] Pasqal - Handle empty environment in qrmi_config

### PR DESCRIPTION
**Pull request by awennersteen**
Handles empty environment in `qrmi_config.json` and add tests for empty env and config file

Do not merge before #38 and #33

### CODE REVIEW

**Comment by MatthieuMoreau0** [comment]
Thanks for the feedback!

From the [test case](https://github.com/qiskit-community/qrmi/pull/39/changes/73e32c80a363273588cf874c2301a40bf45fc5f5#diff-73263414012ace87c1847a0e89b5129df17e8853236164775461f446e7398091R82) implemented by @awennersteen I assumed he wanted to support the case where this environment property was missing. 

I'll let him confirm if there was any particular reason he wanted to support this case. I am personally ok if we want to  enforce the presence of the environment property, but then let's make sure to remove the test case I linked above because it could be misleading @awennersteen.

File: src/pasqal/tests/cloud.rs [hunk 1]
```diff
@@ -73,6 +73,43 @@ async fn is_accessible_attempts_authentication() {
     assert_eq!(qrmi.auth_token, "opaque_token".to_string());
 }
 
+#[test]
+fn read_qrmi_config_env_value_handles_empty_and_missing_environment_key() {
```
Comment (MatthieuMoreau0) [inline src/pasqal/tests/cloud.rs]:
```suggestion
fn read_qrmi_config_env_value_handles_empty_environment_key() {
```

**Review by MatthieuMoreau0 — APPROVED** [review]
Tested with empty config, lgtm. Small documentation nitpick in the tests
